src/test/suite/extension.test.ts

```diff
@@ -54,4 +71,24 @@ suite('Extension Test Suite', () => {
       '7075626c696320636c61737320417070207baa202020207075626c69632073746174696320766f6964206d61696e28537472696e675b5d206172677329207468726f777320457863657074696f6e207ba20202020202020204c6973743c537472696e673e206e616d6573203d204172726179732e61734c697374282244696f676f222c2022436f73746122293ba20202020202020206e616d65732e666f724561636828a202020202020202020202020737472696e67202d3e207ba2020202020202020202020202020202053797374656d2e6f75742e7072696e746c6e28737472696e67293ba2020202020202020202020207da2020202020202020293ba202020207da7d',
     );
   });
+
+  test('Running Display Abstract Syntax Tree Command', async () => {
+    const tabTitle = `${getActiveDocumentFilePath()} - Simple Abstract Syntax Tree`;
+
+    const panel = (await vscode.commands.executeCommand(
+      Commands.DISPLAY_AST,
+    )) as vscode.WebviewPanel;
+
+    assert.strictEqual(panel.title, tabTitle);
+
+    await sleep(5000);
+
+    console.log(panel.webview.html)
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/DiogoTheCoder/make-java-great-again/pullRequest?prid=7068364)
```suggestion
    console.log(panel.webview.html);
```